components/header.tsx

```diff
@@ -17,7 +17,7 @@ const HeaderlessRoutes = [
     '/orders/[orderId]',
     '/orders/[orderId]/labels',
     '/orders/[orderId]/modal',
-    '/product/[productId]',
+    '/product/[productAppExtension]',
```

Comment:
ahh, I think `productId` here is correct - we want to change the name of the repo, not the query param

```suggestion
    '/productAppExtension/[productId]',
```

Reply:
Ohhh okayyyy! Changing this back.